src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -28,7 +28,16 @@
     const getValidationResults = (validator) => {
         const isValid = validator.isValid();
         const validatorName = validator.constructor.name;
-        const result = { isValid, validatorName };
+        const tabErroKeys = validator.fieldsToValidate.reduce((tabsId, value) => {
+            const tabPane = value.item.closest('.tab-pane');
+
+            if (tabPane && value.item.classList.contains('is-invalid')) {
+                tabsId.push(tabPane.id);
```

Comment:
This will produce a little bit complicated structure and you have to loop over it and do some filtering and so.
I would suggest it can be done in this way:
```js
const getInvalidTabs = (validator) => {
    return validator.fieldsToValidate.reduce((invalidTabs, field) => {
        const invalidTabs = field.item.closest('.tab-pane');

        if (tabPane && field.item.classList.contains('is-invalid')) {
            invalidTabs.add(tabPane.id);
        }

        return invalidTabs;
    }, new Set());
}

const invalidTabs = validators.map(getInvalidTabs);
```